plugins/rikki/heroeslounge/components/updatematch/default.htm

```diff
@@ -184,4 +193,47 @@ <h5 class="modal-title">New Game</h5>
     </div>
 </div>
 <!-- End New Game Modal -->
+
+<!-- Start Reschedule Match Modal -->
+<div class="modal fade" role="dialog" id="reschedule_{{__SELF__.match.id}}">
+    <div class="modal-dialog" role="document">
+        <div class="modal-content card-body" id="schedulebox{{__SELF__.match.id}}">
+            <h4 class="card-title text-center">VS <a href="{{ 'team/view' | page({slug:__SELF__.opp.slug})}}"><strong>{{__SELF__.opp.title |upper}}</strong></a></h4>
```

Comment:
Duplication

Reply:
What duplication are you referring to here?

Reply:
Is there no mention of the other teams name elsewhere?